modules/local/SigProfiler/SigProfiler/main.nf

```diff
@@ -0,0 +1,169 @@
+process SIGPROFILER {
+    tag "$meta.id"
+    label "process_high"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'docker://katiad/sigprofiler:version1.0' :
```

Comment:
This will require singularity users to pull and convert the docker container, which in my experience fails half the time.